ubuntu_desktop_installer/lib/l10n/intl_oc.arb

```diff
@@ -0,0 +1,87 @@
+{
+  "@@last_modified": "2020-11-20T17:25:14.036182",
+  "Ubuntu Desktop Installer": "Programa d’installacion del burèu Ubuntu",
+  "@Ubuntu Desktop Installer": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "Go Back": "Tornar",
+  "@Go Back": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "Continue": "Contunhar",
+  "@Continue": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "Restart": "Reaviar",
+  "@Restart": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "Try or install": "Tornar ensajar o installar",
+  "@Try or install": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "Repair installation": "Reparar l’installacion",
+  "@Repair installation": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "Repairing will reinstall all installed software without touching documents or settings.": "La reparacion tornarà installar totes los logicials installats en servant los documents e los paramètres.",
+  "@Repairing will reinstall all installed software without touching documents or settings.": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "Try Ubuntu": "Ensajar Ubuntu",
+  "@Try Ubuntu": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "You can try Ubuntu without making any changes to your computer.": "Podètz ensajar Ubuntu sens aplicar cap de modificacion a vòstre ordenador.",
+  "@You can try Ubuntu without making any changes to your computer.": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "Install Ubuntu": "Installar Ubuntu",
+  "@Install Ubuntu": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "Install Ubuntu alongside (or instead of) your current operating system. This shouldn't take too long.": "Installer Ubuntu à côté (ou en remplacement) de votre système d’exploitation actuel. Ceci ne devrait pas prendre trop longtemps.",
```

Comment:
You forgot to translate this one, it's copy/pasted from the French translations.